src/code/stores/simulation-store.coffee

```diff
@@ -34,8 +35,10 @@ SimulationStore   = Reflux.createStore
     ImportActions, GraphActions]
 
   init: ->
-    defaultUnit = TimeUnits.defaultUnit
-    unitName    = TimeUnits.toString(defaultUnit,true)
+    @defaultUnit = TimeUnits.defaultUnit
+    @unitName    = TimeUnits.toString(@defaultUnit,true)
+    @defaultCollectorUnit = TimeUnits.defaultCollectorUnit
+    @collectorUnitName    = TimeUnits.toString(@defaultCollectorUnit,true)
```

Comment:
The variable `@collectorUnitName` isn't used. Should it be used in `onSetStepUnits` and `onToggledCollectorTo` instead of `@defaultCollectorUnit`?